recipes/openjpeg/recipe.yaml

```diff
@@ -12,8 +12,6 @@ source:
 build:
   number: 0
   script: |
-    # Due to a bug in rattler-build 0.13.0, it is necessary to select appropriate tag
-    git -C "$SRC_DIR" checkout v${{ version }}
```

Comment:
Remove from other recipes too?

https://github.com/neurospin/neuro-forge/blob/f11fe2e0b80a1d21b7b7c2f3157252966c03b5a6/recipes/virtualgl/recipe.yaml#L18-L19

https://github.com/neurospin/neuro-forge/blob/f11fe2e0b80a1d21b7b7c2f3157252966c03b5a6/recipes/libminc/recipe.yaml#L12-L13

Reply:
Probably yes but I did not take time to double check it was fixed in rattler-build. Anyway, something has changed in rattler-build since the `v${{ version }}` substitution in the openjpeg build script was not working anymore. This probably mean that virtualgl is broken too.

Reply:
Ah. It's broken for me too. I think that's my fault, let me have a look.